test/container_test.py

```diff
@@ -1135,11 +1135,11 @@ def test_build_decorator_cls(servicer):
     ret = _run_container(
         servicer,
         "test.supports.functions",
-        "BuildCls.*",
-        inputs=_get_inputs(((), {}), method_name="build1"),
+        # note: builder functions are still run as standalone functions from their class service function
```

Comment:
There are really all sorts of ways that `@build` violate expectations. IMO we should consider whether they're net beneficial. I kind of lean not? Different conversation though.

Reply:
I agree, they are a bit weird. But I do like the concept of keeping related things together in the class, if we can make it more consistent with other class functionality (parameters etc.)